site/docs/getting-started/DesignCards.module.css

```diff
@@ -0,0 +1,40 @@
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  gap: calc(var(--uitk-size-unit) * 4);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+}
+
+.imgContainer {
+  padding: var(--uitk-size-unit);
+  border: 1px solid var(--image-border);
+}
+
+.imgContainer > img {
+  width: 100%;
```

Comment:
I'm not sure this is required as it makes the images quite large on desktop

Reply:
Perhaps using `max-width: 100%` instead would avoid that issue?